medusa/show_updater.py

```diff
@@ -94,6 +95,12 @@ def run(self, force=False):
                                        u'issues while trying to get updates for show {show}. Cause: {cause}',
                                        indexer_name=indexerApi(show.indexer).name, show=show.name, cause=e)
                         continue
+                    except Exception as e:
+                        logger.exception(u'Problem running show_updater, Indexer {indexer_name} seems to be having '
```

Comment:
I'm using logger.exception here, as it will also show a traceback. It should not come to this, but if it does, something is not right, and we'd rather know about it.